deploy/crds/org_v1_che_crd.yaml

```diff
@@ -555,6 +555,11 @@ spec:
                     signed with self-signed cert. So, Che server must be aware of
                     its CA cert to be able to request it. This is disabled by default.
                   type: string
+                serviceHostnames:
+                  description: Use internal host names provided by cluster dns. It
+                    should be usefull to speed internal services traffic. Also it
+                    could avoid some proxy issues. Default value is true.
```

Comment:
`speed` -> `speed up`

Reply:
`Use internal hostnames provided by cluster DNS to speed up the internal services traffic and avoid proxy issues. The default value is `true`.`